src/css/variables.scss

```diff
@@ -24,3 +24,16 @@ $code-font: $code-font-family, serif;
 
 $font-bold: 900;
 
+// Screen Breakpoints
+
+$screen-breakpoint: (
+  xl: 1400px,
+  lg: 1200px,
+  md: 992px,
+  sm: 768px,
+  xs: 576px,
+);
+
+// Z-Index
+
+$z-index-1: 1;
```

Comment:
Just one last thing, instead of naming it `z- index-1`, I'd name it with something that represent the elements that is using it.
Example, `z-index-dropdown` if it's an dropdown. 
This could be `z-index-social-element`.

The reason we name them like that is because if someone needs to add a new `z-index`, it can look at this file and know if it has to be a higher value or lower.

Reply:
I understand and it has sense. Done